### DISCUSSION TITLE
Where can I find the unminified version of the file packages/web/dist/jscad-web.min.js

### DISCUSSION THREAD

**Discussion by Infinity-Forum-Org** [jscad/OpenJSCAD.org#1354] [General]
I'm looking for the unminified version of the code in packages/web/dist/jscad-web.min.js. Does it exist anywhere on Github? It seems to me to be the most important file for modifying the web interface.

**Comment by z3dev** [comment]
@ChristianEngineeringSolutions Welcome!

it really depends on what you are trying to accomplish. But to start, please read the Contributing document, which should provide some good information.

https://github.com/jscad/OpenJSCAD.org/blob/master/CONTRIBUTING.md

**Comment by Infinity-Forum-Org** [comment]
Thank you.
I should have tried reading the file first; it is not so obfuscated that I could not find what I was looking for.
